scripts/th/th-doctor.sh

```diff
@@ -142,38 +144,56 @@ print_framed_text() {
     }
 
     show_more() {
-        # Test Harness Network / DUT
+        # Test Harness DUT Configuration
         th_net_dut_output=$("$TH_DEV_SCRIPTS_DIR/_th-net-dut.sh")
```

Comment:
Maybe this file can be renamed as well
```suggestion
        th_net_dut_output=$("$TH_DEV_SCRIPTS_DIR/_th-dut.sh")
```

Reply:
Done.